rollup-operator/src/synch.js

```diff
@@ -272,6 +274,7 @@ class Synchronizer {
                     const updateFlag = await this._updateEvents([...logsForge,...logsOnChain], lastBatchSaved + 1, targetBlockNumber);
                     if (!updateFlag) continue;
                     lastBatchSaved = await this.getLastBatch();
+                    this.cacheBatchToUpdate = lastBatchSaved;
                 }
```

Comment:
if i understand it correctly here shoudl be `this.retrySameBatch = 0`

Reply:
this should update the next batch that is intended to be synchronized, otherwise `cacheBatchToUpdate` is never updated and it value would be always 0.
I agree that also we should reset `retrySameBatch` variable when a batch is synchronized correctly.